modules/imgproc/src/contours.cpp

```diff
@@ -284,10 +284,13 @@ cvStartFindContours( void* _img, CvMemStorage* storage,
                                           scanner->cinfo_storage );
     }
 
+    assert(step >= 0);
+    assert(size.height >= 1);
```

Comment:
In fact the vanilla `assert` is used, even in this file. But changed mine to `CV_Assert`.